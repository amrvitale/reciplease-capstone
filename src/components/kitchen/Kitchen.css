.myKitchenHero{
    background-image: url(./mykitchen.jpg);
    background-position: center; /* Center the image */
    background-repeat: no-repeat; /* Do not repeat the image */
    background-size: cover; /* Resize the background image to cover the entire container */
}

.myKitchenHeader{
    font-family: 'Amaranth', sans-serif;;
    color: white;
    text-shadow:
    4px 4px 0 black,
    -1px -1px 0 #000,  
     1px -1px 0 #000,
     -1px 1px 0 #000,
      1px 1px 0 #000;
}
h2 {
    font-size: 40px;
}
p {
    font-size: 30px;
    color: rgb(44, 39, 39);
}