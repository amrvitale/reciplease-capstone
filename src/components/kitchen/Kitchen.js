import React from 'react';
import { Link } from 'react-router-dom';
import './Kitchen.css';

/* This component renders the "my kitchen" page, which is where users can click to post a new recipe, or view existing recipes. */

class Kitchen extends React.Component {
    render() {
        return (
            <div className="kitchen">
                <div className="myKitchenHero">
                    <h1 className="myKitchenHeader">My Kitchen</h1>  
                </div>
                <h2 className="start">Let's get cookin'!</h2>
                <p>Welcome to your kitchen. <br />Here you can create a brand new recipe, or view your existing recipes to edit or publish to the Reciplease database.</p>
                <br />
                <Link to='/postrecipe'>
                    <button type="submit">Post New Recipe</button>
                </Link>

                <Link to='/myrecipes'>
                    <button type="submit">View My Recipes</button>
                </Link>
            </div>
        );
    }
}
export default Kitchen;