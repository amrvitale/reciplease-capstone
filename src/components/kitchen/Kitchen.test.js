import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Kitchen from './Kitchen'

describe (`Kitchen component`, () => {
    it ('renders a .Kitchen by default', () => {
        const wrapper = shallow( <Kitchen />)
        expect (toJson(wrapper)).toMatchSnapshot()
    })
});