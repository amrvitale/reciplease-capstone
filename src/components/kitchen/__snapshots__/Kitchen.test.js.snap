// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Kitchen component renders a .Kitchen by default 1`] = `
<div
  className="kitchen"
>
  <div
    className="myKitchenHero"
  >
    <h1
      className="myKitchenHeader"
    >
      My Kitchen
    </h1>
  </div>
  <h2
    className="start"
  >
    Let's get cookin'!
  </h2>
  <p>
    Welcome to your kitchen. 
    <br />
    Here you can create a brand new recipe, or view your existing recipes to edit or publish to the Reciplease database.
  </p>
  <br />
  <Link
    to="/postrecipe"
  >
    <button
      type="submit"
    >
      Post New Recipe
    </button>
  </Link>
  <Link
    to="/myrecipes"
  >
    <button
      type="submit"
    >
      View My Recipes
    </button>
  </Link>
</div>
`;
